src/product-and-services/index.md

```diff
@@ -37,4 +37,4 @@ Our current capacity is in the areas of:
 * quality assurance/testing
 * senior support, consulting and mentoring
 
- <p>💌 Contact us at <a href="mailto:hello@zinc.coop">hello@zinc.coop</a> to discuss how we can help you.</p>
+ <p>💌 <a href="mailto:hello@zinc.coop">Contact us</a> to talk more.</p>
```

Comment:
Can you clarify how this is a fix? Is it just because it's shorter? 

If that's the case, I would say `"Email us at hello@zinc.coop to talk more"` so that people can just read the email address.

Reply:
Correction: this PR reverts a change I made in https://github.com/zinc-collective/www.zinc.coop/pull/105.

<img width="1050" alt="Screenshot 2023-06-11 at 8 55 44 PM" src="https://github.com/zinc-collective/www.zinc.coop/assets/16140873/075524aa-6034-45b2-b0f4-3e87335a9e12">

Reply:
fixed in https://github.com/zinc-collective/www.zinc.coop/pull/106/commits/8e9f966866338c46746ef1fa46bf123c527b8539
![fix](https://github.com/zinc-collective/www.zinc.coop/assets/16140873/daedba16-8f8c-4543-b7b4-ddb7b91b58ea)